test/WebJobs.Script.Tests/Controllers/Admin/SwaggerControllerTests.cs

```diff
@@ -0,0 +1,168 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.ObjectModel;
+using System.Diagnostics;
+using System.Net;
+using System.Net.Http;
+using System.Reflection;
+using System.Threading.Tasks;
+using System.Web.Http.Results;
+using Microsoft.Azure.WebJobs.Script.Config;
+using Microsoft.Azure.WebJobs.Script.Description;
+using Microsoft.Azure.WebJobs.Script.WebHost;
+using Microsoft.Azure.WebJobs.Script.WebHost.Controllers;
+using Microsoft.Azure.WebJobs.Script.WebHost.Filters;
+using Moq;
+using Newtonsoft.Json.Linq;
+using WebJobs.Script.Tests;
+using Xunit;
+
+namespace Microsoft.Azure.WebJobs.Script.Tests.Controllers.Admin
+{
+    public class SwaggerControllerTests
+    {
+        private readonly ScriptSettingsManager _settingsManager;
+        private readonly TempDirectory _secretsDirectory = new TempDirectory();
+        private Mock<ScriptHost> _hostMock;
+        private Mock<WebScriptHostManager> _managerMock;
+        private Collection<FunctionDescriptor> _testFunctions;
+        private SwaggerController _testController;
+        private Mock<ISwaggerDocumentManager> _swaggerDocumentManagerMock;
+        private ScriptHostConfiguration _config = new ScriptHostConfiguration();
+
+        public SwaggerControllerTests()
+        {
+            _settingsManager = ScriptSettingsManager.Instance;
+            _testFunctions = new Collection<FunctionDescriptor>();
+            var environment = new NullScriptHostEnvironment();
+            _hostMock = new Mock<ScriptHost>(MockBehavior.Strict, new object[] { environment, _config, null });
+            _hostMock.Setup(p => p.Functions).Returns(_testFunctions);
+            _hostMock.Object.ScriptConfig.SwaggerEnabled = true;
+
+            WebHostSettings settings = new WebHostSettings();
+            settings.SecretsPath = _secretsDirectory.Path;
+            _managerMock = new Mock<WebScriptHostManager>(MockBehavior.Strict, new object[] { _config, new TestSecretManagerFactory(), _settingsManager, settings });
+            _managerMock.SetupGet(p => p.Instance).Returns(_hostMock.Object);
+            _swaggerDocumentManagerMock = new Mock<ISwaggerDocumentManager>(MockBehavior.Strict);
+            var traceWriter = new TestTraceWriter(TraceLevel.Verbose);
+            _testController = new SwaggerController(_swaggerDocumentManagerMock.Object, _managerMock.Object, traceWriter);            
+        }
+
+        [Fact]
+        public void HasAuthorizationLevelAttribute()
+        {
+            AuthorizationLevelAttribute attribute = typeof(SwaggerController).GetCustomAttribute<AuthorizationLevelAttribute>();
```

Comment:
This test should be updated to use the new attribute and validate that the expected key name was used.